src/pages/superadmin/tableComponent/interfaces.ts

```diff
@@ -16,3 +16,28 @@ export type Bounty = {
   assignee_alias: string;
   paid: boolean;
 };
+
+export interface Person {
+  id: number;
+  unique_name: string;
+  owner_pubkey: string;
+  uuid: string;
+  owner_alias: string;
+  description: string;
+  img: string;
+  tags: string[];
+  pubkey?: string;
+  photo_url?: string;
+  alias?: string;
+  route_hint?: string;
+  owner_route_hint?: string;
+  owner_contact_key?: string;
+  price_to_meet: number;

```

Comment:
> @AbdulWahab3181 Just import the Person interface from main.ts, instead of duplicating it.

Nice work @AbdulWahab3181 just fix this, so I can merge.

Reply:
@elraphty Thanks, Addressed